#55 add cypress to automatic checks+optimizations (#81)
timmyichen/dev-directory: README.md

We use [cypress](https://docs.cypress.io/guides/overview/why-cypress) for UI testing. Cypress requires the app to be running for testing.

> To run tests manually,
> * Execute `npm run cypress:open`
> * Select E2E testing
> * Choose a browser
> * run the requisite test spec from the list.

Reviewer: nit: Punctuation, capitalization, simplified grammar
```suggestion
To run tests manually:
* Execute `npm run cypress:open`
* Select E2E testing
* Choose a browser
* Select the test spec you wish to run
```
